tests/EvidenceForTest.py

```diff
@@ -28,7 +27,14 @@ def tearDown(self):
         PyOpenWorm.disconnect(self.conn)
 
     def test_evidence_for(self):
-        qctx = Context()
-        qctx(Neuron)('AVAL').innexin('UNC-7')
-        ev_iterable = evidence_for(qctx, self.conn, Evidence, Context)
-        self.assertTrue((len(ev_iterable) != 0))
\ No newline at end of file
+        c1 = Context()
```

Comment:
Not quite -- please have this TestCase subclass DataTestTemplate._DataTest and you'll see this test still fails.

Reply:
Sorry: also remove the setUp/tearDown implementations -- the point is that you are using the OpenWormData database, but should not be.